lib/offsite_payments/integrations/universal.rb

```diff
@@ -50,6 +50,11 @@ def initialize(order, account, options = {})
           @forward_url = options[:forward_url]
           @key = options[:credential2]
           @currency = options[:currency]
+
+          # Some confusion was caused by the fact the Universal Offsite Dev Kit uses credential3 to store the POST url for testing.
+          # The condition checked will ensure that x_credential3 is not sent when using the dev kit.
```

Comment:
I think `checked` is grammatically correct in this instance but i can make more concise.